### DISCUSSION TITLE
Welcome to Podman Discussions!

### DISCUSSION THREAD

**Discussion by TomSweeneyRedHat** [containers/podman#9042] [General]
![image](https://user-images.githubusercontent.com/25613829/105226220-b992e580-5b2d-11eb-8d4d-11c47eb3c975.png)

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the Podman community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪 and we expect participants to follow the [Podman Project Community Code of Conduct](https://github.com/containers/podman/blob/master/CODE-OF-CONDUCT.md).

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vrothberg** [comment]
Let's not be shy :)

I am Valentin, and I help build and maintain Podman and its sibling projects Buildah, Skopeo and CRI-O and the underlying libraries.  Currently, I live in France and am available during normal EU working hours.  You can find me on Freenode's IRC #podman channel as `vrothberg`.  Feel free to ping me on IRC and also here on GitHub any time.

It's the first time for me using GitHub discussions and my hopes are high that together we can turn it into a place to share ideas and knowledge, create helpful FAQs and How-Tos and have fun.

I think we need to fill this place with life to kick things off.  Maybe we can move some of the tutorials over here, the rootless .md file and potentially others.  Let's have some more @containers/podman-maintainers say hello before :)

**Comment by saschagrunert** [comment]
Hey folks! :wave: 

I'm Sascha, helping to maintain Podman and surrounding projects as well. I'm right now more focused on the Kubernetes side of things, which also includes CRI-O and the shared libraries within the containers/ project. Beside that, I'm working within Kubernetes SIG Release and Node to support the community there.

Feel free to reach out anytime if you have any questions or comments, I'm available on [Kubernetes Slack (@sascha)](https://slack.k8s.io/) or Freenode's #podman, #cri-o or #containers channels as `saschagrunert`. Since I live in Germany I'll be mostly around during the usual CET working hours.

See y'all later! :upside_down_face:

**Comment by TomSweeneyRedHat** [comment]
Howdy!

My turn!  I'm Tom Sweeney, one of the folks that works on a number of the
container technologies here at Red Hat.   My primary focus is Buildah,
followed by Podman, and I also dabble in CRI-O, Docker, and containerd. 
On occasion, I stick my nose into containers/storage and containers/image
and I manage the https://podman.io and https://buildah.io blog sites. 

Away from work, dad of 3, grand-dad of 1 (soon to be 2), and a lifelong Boston Red Sox
fan.  One of my life's regrets is not being able to hit a good curveball.

t

Twitter: @TSweeneyRedHat
GitHub: @TomSweeneyRedHat

**Comment by rhatdan** [comment]
Hello, 
My name is Dan Walsh.  I work on all things containers at the Operating System level.  Been working on these technologies for almost 20 years, and love talking and working on controlling what processes can do on a Linux system.

Feel free to contact me on any of these communications platforms

Live on US East Coast near Boston, but prefer to live in Southern Maine.
Slack/Twitter/Github and most places on the internet rhatdan
IRC dwalsh usually lurking in `#podman`
Email dwalsh@redhat.com

**Comment by baude** [comment]
Greetings earthlings,

My name is Brent and I am on the container runtimes team at Red Hat.  My main focus has been to work on Podman but I also work on some of its related technologies like dnsname, etc.

I live near Harris, MN about 50 minutes north of the Minneapolis and St. Paul area.  I can be contacted during work hours in #podman on freenode as user `baude` or some modification of that.
